cmake-modules/FolderList.cmake

```diff
@@ -2,16 +2,13 @@ macro(GetFolderList project)
     message ("project found ${project}")
     message ("FLAG VALUE : ${FETCH_SOURCE_DEPS}")
     if(${project} STREQUAL CERTIFICATES)
-        DownloadDepVersion(sdk/core azure-core 1.2.0)
-        DownloadDepVersion(sdk/identity azure-identity 1.1.0)
+        DownloadDepVersion(sdk/core azure-core 1.5.0)
     elseif(${project} STREQUAL IDENTITY)
         DownloadDepVersion(sdk/core azure-core 1.2.0)
     elseif(${project} STREQUAL SECRETS)
-        DownloadDepVersion(sdk/core azure-core 1.2.0)
-        DownloadDepVersion(sdk/identity azure-identity 1.1.0)
+        DownloadDepVersion(sdk/core azure-core 1.5.0)
     elseif(${project} STREQUAL KEYS)
-        DownloadDepVersion(sdk/core azure-core 1.2.0)
-        DownloadDepVersion(sdk/identity azure-identity 1.1.0)
```

Comment:
I don't think identity should be removed from here

Reply:
true